<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MM32 Programmer V1.00</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icon/icon/MM32_Logo.ico</normaloff>:/icon/icon/MM32_Logo.ico</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="ProjectTree" name="treeWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
       <widget class="ReadFlash" name="flash" native="true"/>
      </widget>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menucaidan">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="loadProgrammingFile"/>
    <addaction name="saveProgrammingFile"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menu_P">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="newProject"/>
    <addaction name="openProject"/>
    <addaction name="separator"/>
    <addaction name="saveProject"/>
    <addaction name="saveProjectAs"/>
    <addaction name="separator"/>
    <addaction name="closeProject"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>Operate</string>
    </property>
    <addaction name="checkLink"/>
    <addaction name="separator"/>
    <addaction name="chipLink"/>
    <addaction name="separator"/>
    <addaction name="cheakBlank"/>
    <addaction name="erasureChip"/>
    <addaction name="separator"/>
    <addaction name="programmeChip"/>
    <addaction name="pyCode"/>
    <addaction name="separator"/>
    <addaction name="verify"/>
    <addaction name="unlockChip"/>
    <addaction name="separator"/>
    <addaction name="readChip"/>
    <addaction name="writeChip"/>
    <addaction name="separator"/>
    <addaction name="options"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="about"/>
   </widget>
   <addaction name="menucaidan"/>
   <addaction name="menu_P"/>
   <addaction name="menu_O"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="newProject"/>
   <addaction name="openProject"/>
   <addaction name="modifyProject"/>
   <addaction name="saveProject"/>
   <addaction name="separator"/>
   <addaction name="checkLink"/>
   <addaction name="chipLink"/>
   <addaction name="separator"/>
   <addaction name="erasureChip"/>
   <addaction name="programmeChip"/>
   <addaction name="unlockChip"/>
   <addaction name="separator"/>
   <addaction name="readChip"/>
   <addaction name="writeChip"/>
   <addaction name="separator"/>
   <addaction name="loadProgrammingFile"/>
   <addaction name="saveProgrammingFile"/>
  </widget>
  <action name="newProject">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/1.png</normaloff>:/icon/icon/1.png</iconset>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="openProject">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/2.png</normaloff>:/icon/icon/2.png</iconset>
   </property>
   <property name="text">
    <string>Open Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveProject">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/3.png</normaloff>:/icon/icon/3.png</iconset>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveProjectAs">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/4.png</normaloff>:/icon/icon/4.png</iconset>
   </property>
   <property name="text">
    <string>Save Project As...</string>
   </property>
  </action>
  <action name="erasureChip">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/erase.png</normaloff>:/icon/icon/erase.png</iconset>
   </property>
   <property name="text">
    <string>Erase Chip</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="programmeChip">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/program.png</normaloff>:/icon/icon/program.png</iconset>
   </property>
   <property name="text">
    <string>Program</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="unlockChip">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/lock.png</normaloff>:/icon/icon/lock.png</iconset>
   </property>
   <property name="text">
    <string>Unlock Chip</string>
   </property>
  </action>
  <action name="loadProgrammingFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/6.png</normaloff>:/icon/icon/6.png</iconset>
   </property>
   <property name="text">
    <string>Load Data File</string>
   </property>
  </action>
  <action name="saveProgrammingFile">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/5.png</normaloff>:/icon/icon/5.png</iconset>
   </property>
   <property name="text">
    <string>Save Data File</string>
   </property>
  </action>
  <action name="closeProject">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="readChip">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/read.png</normaloff>:/icon/icon/read.png</iconset>
   </property>
   <property name="text">
    <string>Read Chip</string>
   </property>
  </action>
  <action name="writeChip">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/write.png</normaloff>:/icon/icon/write.png</iconset>
   </property>
   <property name="text">
    <string>Write Chip</string>
   </property>
  </action>
  <action name="options">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="checkLink">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/disConnect.png</normaloff>:/icon/icon/disConnect.png</iconset>
   </property>
   <property name="text">
    <string>Programmer Connect</string>
   </property>
  </action>
  <action name="chipLink">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/chip.png</normaloff>:/icon/icon/chip.png</iconset>
   </property>
   <property name="text">
    <string>Chip Link</string>
   </property>
  </action>
  <action name="cheakBlank">
   <property name="text">
    <string>Cheak Blank</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="pyCode">
   <property name="text">
    <string>Program and Verify</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="verify">
   <property name="text">
    <string>Verify</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="modifyProject">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/option.png</normaloff>:/icon/icon/option.png</iconset>
   </property>
   <property name="text">
    <string>modify Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProjectTree</class>
   <extends>QTreeWidget</extends>
   <header location="global">interface.project_tree</header>
  </customwidget>
  <customwidget>
   <class>ReadFlash</class>
   <extends>QWidget</extends>
   <header location="global">interface.read_flash</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
